### ISSUE TITLE
Use #include instead of @import?

### ISSUE THREAD

**Issue by mmontag** [mentos1386/webpack-glsl-loader#5]
See https://github.com/grieve/webpack-glsl-loader/pull/6. 

I believe it was a mistake to call this behavior "import", but maybe that's just me...

**Comment by mmontag** [comment]
Actually, I wonder if this should be implemented through comments, since it is not part of GLSL spec, and not supported by GLSL tools (see https://github.com/mentos1386/ts-shader-loader/issues/3).

For example,

```
//#include "./logdepth_pre_vert.glsl";
...
```

instead of  
```
@import ./logdepth_pre_vert;
...
```

Altogether, I'm suggesting four changes:
1. hidden behind comment, so it's transparent to other GLSL tooling.
2. the file path uses quotes.
3. the file path can include .glsl suffix. (`Error: Can't resolve './logdepth_pre_vert.glsl.glsl'` - this is silly)
4. the syntax is `include` instead of `import` so it's clear this is a preprocessor-like dumb string concatenation, not a symbol import.

**Comment by mentos1386** [comment]
@mmontag good idea. I'll pull the changes from https://github.com/grieve/webpack-glsl-loader/pull/6.

**Comment by mentos1386** [comment]
This was released in 2.0.1 https://www.npmjs.com/package/ts-shader-loader/v/2.0.1
